tests/journald_tests.cpp

```diff
@@ -270,15 +290,30 @@ TEST_F(JournaldLoggerTest, ROOT_LogToJournald)
       statusRunning->executor_id().value(),
       "runs",
       "latest");
-
   ASSERT_TRUE(os::exists(sandboxDirectory));
 
-  string stdoutPath = path::join(sandboxDirectory, "stdout");
-  ASSERT_TRUE(os::exists(stdoutPath));
+  std::string stdoutPath = path::join(sandboxDirectory, "stdout");
+
+  if (GetParam() == "journald") {
+    // Check that the sandbox was *not* written to.
+    // TODO(josephw): This file exists as other parts of the agent will
+    // create it.  We should invert this assertion when we fix this in Mesos.
```

Comment:
The fetcher and docker containerizer create the empty files:
* https://github.com/apache/mesos/blob/1.1.x/src/slave/containerizer/fetcher.cpp#L740-L767
* https://github.com/apache/mesos/blob/1.1.x/src/slave/containerizer/docker.cpp#L251-L263

As for a JIRA... https://issues.apache.org/jira/browse/MESOS-6843